tests/bifrost/workflow_test.py

```diff
@@ -125,3 +145,131 @@ def test_simulation_workflow_produces_the_same_data_as_before(
         )
 
     sc.testing.assert_allclose(energy_data.bins.data, expected.bins.data)
+
+
+class TestBifrostQCutWorkflow:
```

Comment:
Please move to a separate file to match the file structure of the package and make it easier to find tests.